.github/workflows/ci-backend.yml

```diff
@@ -70,6 +70,7 @@ jobs:
                         - 'ee/**/*'
                         - 'common/hogvm/**/*'
                         - 'posthog/**/*'
+                        - 'dags/**'
```

Comment:
Looks like I missed this here: https://github.com/PostHog/posthog/pull/28753

Seems like things would just be easier if this lived in the existing `posthog.*` package though, between this and [hacks around fixture discovery](https://github.com/PostHog/posthog/blob/b60dce3d5334ad407ac19905c390414e2f231f6d/dags/tests/conftest.py#L1-L2), it feels like cutting across the grain for little benefit, especially since that package needs to be importable anyway and `dagster` is part of the  base requirements